### PR TITLE
Feature coin convertor

### PR DESCRIPTION
**Pull request by Chase-T-White**
- Adding coin convertor functionality
- Adding coin convertor chart
- Adding time scale buttons for coin convertor chart

### CODE REVIEW

File: app/ui/home/TimeScaleSelection.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  onClickFunction: any;
+}) => {
+  return (
+    <ul className="w-max flex gap-2 p-1 text-sm text-darkTheme-white-500 bg-dark-purple-700 rounded-md">
+      <button
+        value={1}
+        className={`px-5 py-2 rounded-md ${
+          timeScale === 1 ? "active-button" : ""
+        }`}
+        onClick={(e) => handleClick(e)}
```
Comment (yoloOnTheBattlefield) [inline app/ui/home/TimeScaleSelection.tsx]: onClick={handleClick}

File: app/ui/home/convertor/CoinConvertorChartSection.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            </div>
+          )}
+        </div>
+        <ul className="w-max flex gap-2 p-1 text-sm text-darkTheme-white-500 bg-dark-purple-700 rounded-md">
+          <button
+            value={1}
+            className={`px-5 py-2 rounded-md ${
+              timeScale === 1 ? "active-button" : ""
+            }`}
+            onClick={(e) => handleClick(e)}
```
Comment (yoloOnTheBattlefield) [inline app/ui/home/convertor/CoinConvertorChartSection.tsx]: same here

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: app/ui/home/TimeScaleSelection.tsx (hunk 1 above)
Comment (Chase-T-White) [reply app/ui/home/TimeScaleSelection.tsx]: issue has been addressed

File: app/ui/home/convertor/CoinConvertorChartSection.tsx (hunk 2 above)
Comment (Chase-T-White) [reply app/ui/home/convertor/CoinConvertorChartSection.tsx]: issue has been addressed

**Pull request merged**